manifests/crd.yaml

```diff
@@ -29,10 +29,6 @@ spec:
         type: string
         description: Status of the main pod
         jsonPath: .status.mainPod.status.phase
-      - name: Age
-        type: date
-        description: Creation timestamp of the JupyterServer
-        jsonPath: .metadata.creationTimestamp
```

Comment:
Are you sure this is OK? See https://github.com/SwissDataScienceCenter/amalthea/pull/723

Reply:
You are right I will put that back. I did not see Alessandro added that in the wrong file.